packages/babel-traverse/src/path/modification.js

```diff
@@ -142,10 +135,7 @@ export function _insertAfter(nodes, shouldRequeue = false) {
     ) {
       nodes.unshift(this.node);
     }
-    this._replaceWith(t.blockStatement(nodes));
-    if (shouldRequeue) {
-      this.requeue();
-    }
+    this.replaceWith(t.blockStatement(nodes));
```

Comment:
I thought of calling `insertAfter` directly too but assumed the reason this piece used `_insertAfter` is for performance reason, i.e. avoid double queuing when this is invoked by `replaceWithMultiple`. If that's not a problem (and there's no other reason for this piece to use `_insertAfter` in the first place), I like this change as it's simpler.